lg_stats/msg/Session.msg

```diff
@@ -0,0 +1,5 @@
+string mode
```

Comment:
lg_stats is going to be writing influxdb info. It's based on two types of data - events and time series. I think this message type should be just named "Event" and have a json string attribute that will be inserted into influxdb via agent. 

I'm totally not sure about all of this tho.